scripts/list_boards.py

```diff
@@ -20,9 +21,11 @@
 except ImportError:
     from yaml import SafeLoader
 
-BOARD_SCHEMA_PATH = str(Path(__file__).parent / 'schemas' / 'board-schema.yml')
+BOARD_SCHEMA_PATH = str(Path(__file__).parent / 'schemas' / 'board-schema.json')
 with open(BOARD_SCHEMA_PATH, 'r') as f:
-    board_schema = yaml.load(f.read(), Loader=SafeLoader)
+    board_schema = json.load(f)
+
+board_validator = jsonschema.Draft202012Validator(board_schema)
```

Comment:
The schema reference is part of the schema itself, prefer to try to load it from the file

```suggestion
board_validator = jsonschema.validators.validator_for(board_schema)
```